### ISSUE TITLE
Keychain error when initiated Nuance from lock screen

### ISSUE THREAD

**Issue by narendra-r** [NuanceDev/speechkit-ios-sample-swift#2]
HI i need to use nuance on lock screen call, I enabled the keychain capapbility for my bundle id too but got this error,
*** Assertion failure in -[KeychainManager writeToKeychain], /Users/martinservant/Work/NMSP/NMSP5.3_multi_channel_c/sdk/c/trunk/source/oem/iphone/src/KeychainManager.m:293

Can you please help me to resolve this issue.

**Comment by jkgneu12** [comment]
@narendra-r You enabled Keychain Sharing under Capabilities?

**Comment by narendra-r** [comment]
Yes @jkgneu12 , i enabled  Keychain Sharing under Capabilities and it works well when app is open. The only issue happened when the app needs to take the call from the lock screen.

**Comment by narendra-r** [comment]
@jkgneu12 can you please help on this?
